fontc/src/lib.rs

```diff
@@ -3400,4 +3410,26 @@ mod tests {
             ]
         );
     }
+
+    #[test]
```

Comment:
I suggest adding a test that just says some other source doesn't produce a meta table. And maybe another for empty meta where the glyphs file has "meta Table"  but nothing in it.